digest/generated/digest.json

```diff
@@ -0,0 +1,105 @@
+{
+  "schema_version" : "1.0.0",
+  "pipeline_name" : "digest",
+  "externs" : [
+    {
+      "name" : "Digest",
+      "id" : "0x05000000",
+      "permissions" : "0x19b6",
+      "instances" : [
+        {
+          "inst_name" : "Ingress_Deparser.digest_inst",
+          "inst_id" : 1,
+          "params" : [
+            {
+              "id" : 1,
+              "name" : "index",
+              "type" : "bit32",
+              "attr" : "tc_key",
+              "bitwidth" : 32
+            },
+            {
+              "id" : 2,
+              "name" : "srcAddr",
+              "type" : "macaddr",
+              "attr" : "param",
+              "bitwidth" : 48
+            },
+            {
+              "id" : 3,
+              "name" : "ingress_port",
+              "type" : "dev",
+              "attr" : "param",
+              "bitwidth" : 32
+            },
+            {
+              "id" : 4,
+              "name" : "drop_reason",
+              "type" : "bit8",
+              "attr" : "param",
+              "bitwidth" : 8
+            }
+          ]
+        }
```

Comment:
We need the drop reason not as a number but as a name for human consumption. Maybe i am missing something.

Reply:
We discussed offline and what we'd like would be for the compiler to emit in the JSON the mapping from integer to corresponding enum string (`parser_rejected` or `table_miss`)